### PR TITLE
[Bugfix] Removed the related UITests on control views in setup screen

### PR DESCRIPTION
**Pull request by kevinyulu**
## Purpose
Removed the related UITests on control views in setup screen

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
* Open Xcode and launch UCUITest 

## What to Check
Verify that the following are valid
* verify if all the UXUItests pass.

### CODE REVIEW

**Review by palatter — DISMISSED** [review]
Looks good.

File: AzureCommunicationUI/AzureCommunicationUIDemoAppUITests/AzureCommunicationUIDemoAppLaunchTests.swift [hunk 1]
```diff
@@ -7,20 +7,18 @@ import XCTest
...
-    func testCallCompositeLaunch() {
-        tapInterfaceFor(.uiKit)
+    func testLaunch() {
+        let app = XCUIApplication()
+        app.launch()
+
+        let attachment = XCTAttachment(screenshot: app.screenshot())
+        attachment.name = "Launch Screen"
+        attachment.lifetime = .keepAlways
+        add(attachment)
```
Comment (AmyL219) [inline AzureCommunicationUI/AzureCommunicationUIDemoAppUITests/AzureCommunicationUIDemoAppLaunchTests.swift:17]: What's the attachment purpose for? It only work in the `testLaunch` function, correct?

**Review by AmyL219 — APPROVED** [review]

File: AzureCommunicationUI/AzureCommunicationUIDemoAppUITests/AzureCommunicationUIDemoAppLaunchTests.swift (hunk 1 above)
Comment (kevinyulu) [reply AzureCommunicationUI/AzureCommunicationUIDemoAppUITests/AzureCommunicationUIDemoAppLaunchTests.swift:17]: I think this method existed before in the other test, I moved this function to this test class.

Comment (kevinyulu) [reply AzureCommunicationUI/AzureCommunicationUIDemoAppUITests/AzureCommunicationUIDemoAppLaunchTests.swift:17]: This is to take a screenshot after the screen was launched. we can remove if this is not needed.

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**